cmsplugin_articles_ai/models/plugin_models.py

```diff
@@ -48,6 +49,17 @@ class ArticleListPlugin(CMSPlugin):
         default=5,
         verbose_name=_("amount of articles"),
     )
+    language_filter = LanguageField(
+        verbose_name=_("language filter"),
+        default="",
+        blank=True,
+        help_text=_(
```

Comment:
The translatable text would include lots of spaces all around. Please spell this (as ugly as it might look) like this:

```
help_text=_(
    "Select a language if you want to list only articles written in specific "
    "language. If you don't select a language, the listing includes all languages."
)
```